package javacore.Exececoes.exception.dominio;

import java.io.FileNotFoundException;
import java.io.IOException;

public class Funcionario extends Pessoa {
    public void salvar() throws LoginInvalidoException,ArithmeticException {
        System.out.println("Testando 1,2,3");
    }
}
